packages/function-tree-firebase-admin/src/Provider.js

```diff
@@ -24,43 +24,23 @@ function FirebaseAdminProvider (options = {}, customFirebaseInstance) {
         return firebase.database().ref(path).push().key
       },
       deleteUser (uid) {
-        return new Promise((resolve, reject) => {
-          firebase.auth().deleteUser(uid)
-            .then(() => resolve())
-            .catch((error) => reject({error}))
-        })
+        return firebase.auth().deleteUser(uid)
       },
       remove (path) {
         const ref = firebase.database().ref(path)
 
-        return new Promise((resolve, reject) => {
-          ref.remove()
-            .then(() => resolve())
-            .catch((error) => reject({error: error.message}))
-        })
+        return ref.remove().then(() => undefined)
```

Comment:
Want to be explicit that we want to return undefined, so others do not think that the `then` can be removed